invenio_rdm_records/records/jsonschemas/records/parent-v1.0.0.json

```diff
@@ -40,7 +40,7 @@
             "properties": {
               "subject": {
                 "type": "string",
-                "enum": ["user", "role", "sysrole"]
+                "enum": ["user", "role", "system_role"]
```

Comment:
because up until now the grants were actually not dumped, this property was actually unused... so i guess this is a backwards-compatible change? :eyes:

Reply:
this comment didn't age very well; there's so many changes now that i think a new version is justified.